pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -430,6 +437,9 @@ func (c AdHocCheck) Type() CheckType {
 	case c.Settings.Grpc != nil:
 		return CheckTypeGrpc
 
+	case c.Settings.Browser != nil:
+		return CheckTypeScripted
+
```

Comment:
Should this be 

```suggestion
	case c.Settings.Browser != nil:
		return CheckTypeBrowser

```

instead?